### PR TITLE
Broadcast

### PR DESCRIPTION
**Pull request by Synconix**
**Automatic Broadcasting System**

**What it will look like -** https://imgur.com/x2bJrSG

_**Reply or tag me on #contributing in discord if you want me to change something (Such as the color, time, message, etc.)**_

### CODE REVIEW

**Comment by BennyDoesTheStuff** [comment]
**BTW:**
Since Matrix isn't staff anymore and he removed some of the stuff he made for Warzone, one of those things removed from the server was the /discord and /mode commands, so you may wanna change the announcement message to reflect these changes.

**Comment by Synconix** [comment]
**Edited** Thanks Benny!

**Comment by BennyDoesTheStuff** [comment]
#122

**Comment by Aerh** [comment]
I don't think you can extend the `JavaPlugin` class more that once? O.o

File: TGM/src/main/java/network/warzone/tgm/command/CycleCommands.java [hunk 1]
```diff
@@ -304,6 +304,11 @@ public static void next(CommandContext cmd, CommandSender sender) {
         sender.sendMessage(ChatColor.YELLOW + "Next Map: " + ChatColor.GRAY + TGM.get().getMatchManager().getMapRotation().getNext().getMapInfo().getName());
     }
     
+    @Command(aliases = {"website", "web", "site"}, desc = "View warzone official website")
+    public static void next(CommandContext cmd, CommandSender sender) {
+        sender.sendMessage(ChatColor.AQUA + "View stats, leaderboard, store and more all at https://warz.one/ ");
+    }
+
```
Comment (DeternoMC) [inline TGM/src/main/java/network/warzone/tgm/command/CycleCommands.java]: You should really be creating different pull requests for different patches.

**Comment by DeternoMC** [comment]
check #182 for the /website and /buy. Remove it from this pull request :)

**Comment by Synconix** [comment]
**Done 👍**

**Comment by Trophonix** [comment]
You can't put another plugin into the code of an existing plugin. That class will never even be loaded by bukkit.
